aiogram/dispatcher/handler.py

```diff
@@ -23,11 +23,11 @@ class CancelHandler(Exception):
 
 
 def _get_spec(func: callable):
+    wrapped_function = func
     while hasattr(func, '__wrapped__'):  # Try to resolve decorated callbacks
         func = func.__wrapped__
-
     spec = inspect.getfullargspec(func)
-    return spec, func
+    return spec, wrapped_function
```

Comment:
In this case maybe don't need to return the handler from this function because decorated handler is already accessible in place from which _get_spec is called

Reply:
Yes, thank you! Didn't think about it, that's true. Updated in [d2d4928](https://github.com/aiogram/aiogram/pull/180/commits/d2d49282f55544886a38a577d6692c0d7349115c)